distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -1,43 +1,25 @@
+=== Solr Catalog Types
+The ${branding} supports 3 Solr Catalog Types:
 
-==== Catalog Solr Embedded Provider
+. Solr Catalog Embedded Provider - included with the ${branding}, useful for testing and demonstration pruposes
+. Solr Catalog External Provider - single Solr Server instance
+. SolrCloud - cluster of standalone Solr Servers
```

Comment:
`SolrCloud` to `Solr Catalog Cloud Provider` for consistency with the previous two.  This should be applied to all files that use `SolrCloud`.